src/Point/Point.csproj

```diff
@@ -56,6 +56,7 @@
   <Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish">
     <!-- As part of publishing, ensure the JS resources are freshly built in production mode -->
     <Exec WorkingDirectory="$(SpaRoot)" Command="npm install" />
+    <Exec WorkingDirectory="$(SpaRoot)" Command="npm i --save-dev @types/google-protobuf" />

```

Comment:
what is this about? why isn't it enough if we add @types/google-protobuf to our dev dependencies